src/utils/firebase_list_observable.ts

```diff
@@ -40,9 +40,14 @@ export class FirebaseListObservable<T> extends Observable<T> {
     });
   }
 
-  remove(item:FirebaseOperation): Promise<void> {
+  remove(item:FirebaseOperation = null): Promise<void> {
     // TODO: remove override when typings are updated to include
     // remove() returning a promise.
+    
+    // if no item parameter is provided, remove the whole list
+    if (!item) {
```

Comment:
We should support it because the Firebase SDK does. The Firebase SDK coerces the `number` to a `string` for `.child()`.

Below is totally valid

```ts
const ref = new Firebase('<my-firebase-db');
const zeroChild = ref.child(0);
zeroChild.remove();
```